src/vhdlConvertor/designFileParser.cpp

```diff
@@ -124,16 +125,37 @@ void VhdlDesignFileParser::visitContext_item(
 	// ;
 	auto l = ctx->library_clause();
 	if (l) {
-		NotImplementedLogger::print(
-				"DesignFileParser.visitContext_item - library_clause", l);
-		return; //libraries are ignored
+		visitLibrary_clause(l);
 	}
 	auto u = ctx->use_clause();
 	if (u) {
 		visitUse_clause(u, context.objs);
 	}
 }
 
+void VhdlDesignFileParser::visitLibrary_clause(
+		vhdlParser::Library_clauseContext *ctx) {
+	//rhinton:: I didn't create a method to visit a logical name 
```

Comment:
It would be best if we have some method which extract the vector<string> from identifier_list and use it also there https://github.com/Nic30/hdlConvertor/blob/c546f023ce1133abffadc8153794d38fdb9306ca/src/vhdlConvertor/interfaceParser.cpp#L27 (Which I believe is only place where identifier_list is used), However identifier_list rule is very simple, so I see it as very low priority task.

Reply:
Interesting.  This seems fairly easy to break out.  I would expect to see the same kind of list in a signal declaration and variable declaration, at least.  I'll take a look.

Reply:
I did break this out into a separate method.  It's only a few lines of code.  In fact, `identifier_list` is used everywhere!  

But in all the places I looked, the existing code iterates over the identifier list creating new AST objects.  So using a method to transform the identifier list to a vector would result in the existing code iterating over a vector instead of over the identifier list -- adding a line of code essentially for no purpose.  So I removed this function.  I can rewrite it if you like.